lib/services/services_lsb.c

```diff
@@ -238,8 +238,6 @@ services__get_lsb_metadata(const char *type, char **output)
     lsb_meta_helper_free_value(should_stop);
     lsb_meta_helper_free_value(default_start);
     lsb_meta_helper_free_value(default_stop);
-
-    crm_trace("Created fake metadata: %zu", strlen(*output));
```

Comment:
Can you elaborate on why we can't use `%z` until we require libqb 1.0.1?

Reply:
crm_trace() etc. don't use C library functions, they use libqb functions, and libqb didn't support %z/%j/%t until 1.0.1

Reply:
I knew they used libqb. Silly me, I assumed the libqb functions that use `printf`-style specifiers called one of the `printf`-family functions......